viewmaster/link.py

```
@@ -22,17 +37,21 @@
 #     LOG_ROOT_PREFIX_ = '/Library/WebServer/Logs/webapps/'
 ################################################################################
 
-from viewmaster_config import *
+from .viewmaster_config import *
 
 LOGNAME = LOGNAME.replace('viewmaster', 'link')
 LOGGER = pdslogger.PdsLogger(LOGNAME, limits={'info': -1, 'normal': -1},
                                       pid=True)
 
 LOG_FILE = LOG_ROOT_PREFIX_ + 'link.log'
```

Ruff (0.14.5)

`LOG_ROOT_PREFIX_` may be undefined, or defined from star imports

(F405)